.devcontainer/Dockerfile

```
@@ -95,30 +89,18 @@ RUN useradd trickfire \
     # Give it a directory in /home
     --create-home \
     # Don't make a giant log file for login data, we don't care about it
-    --no-log-init \
-    # Allow the user to access video cameras.
-    -G video,zed
+    --no-log-init
 
-# Copy all the bash customizations over to the user.
+# Copy all the bash customizations over to the user
 COPY .devcontainer/trickfire.bashrc /home/trickfire/.bashrc
 
-# Give sudo access.
+# Give sudo access
 RUN echo "trickfire ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/user
 
-# Fix permissions.
+# Fix permissions
 RUN rosdep fix-permissions && \
     chown -R trickfire:trickfire /usr/local/lib/python3.10/dist-packages && \
     chmod -R a+rX /usr/local/lib/python3.10/dist-packages
 
-# Change to the non-root user.
+# Change to the non-root user
 USER trickfire
-
-RUN sudo curl https://packages.osrfoundation.org/gazebo.gpg --output /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
-RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] https://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
-RUN sudo apt-get update && sudo apt-get install -y ignition-fortress
-
-# TODO: We might want another stage to minimize the image on the rover.
-
-ENV DISPLAY=:0
```

```suggestion
ENV DISPLAY=:0
```
i think we need this somewhere in the Dockerfile to be able to see GUI from the container

---

Because the macos version doesnt work on linux, might not work on windows, should I update the docker container to try to adjust based on operating system and/or system architecture?